### PR TITLE
Bump moby to 1ffa10ee2630917e4f2a88f5e4daf34d700eaa1f

### PR DESCRIPTION
**Pull request by thaJeztah**
Full diff: https://github.com/moby/moby/compare/14ce1f1cf48e9859223c6311de58aec4dc0f046c...1ffa10ee2630917e4f2a88f5e4daf34d700eaa1f

### CODE REVIEW

File: vendor/github.com/docker/docker/api/types/types.go [hunk 1]
```diff
@@ -327,7 +328,7 @@ type ContainerJSONBase struct {
 	Name            string
 	RestartCount    int
 	Driver          string
-	Platform        string
+	Platform        specs.Platform
```
Comment (thaJeztah) [inline vendor/github.com/docker/docker/api/types/types.go]: This change will be reverted again after https://github.com/moby/moby/pull/35144 is merged

**Comment by jhowardmsft** [comment]
No, don't take this until the fix is in, and re-vendor accordingly.

**Comment by thaJeztah** [comment]
Updated; now includes https://github.com/moby/moby/pull/35144

**Comment by jhowardmsft** [comment]
LGTM (not a maintainer in this repo)

**Comment by thaJeztah** [comment]
ping @vdemeester

**Pull request merged**